assets/app/view/game/corporation.rb

```diff
@@ -48,6 +48,13 @@ def render
         unless @corporation.minor?
           children << render_shares
           children << h(Companies, owner: @corporation, game: @game) if @corporation.companies.any?
+          if @corporation.can_par?(@game.current_entity)
```

Comment:
2 logic in similar places, maybe in the game class we can have a method

def par_prices_for(corporation)
  par_prices.select { ... }

that should clean this up a bit